src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputFormattedJavaDocTagContinuationIndentation.java

```diff
@@ -0,0 +1,298 @@
+package com.google.checkstyle.test.chapter7javadoc.rule713atclauses;
+
+import java.io.Serializable;
+
+/**
+ * Some javadoc.
+ *
+ * @since Some javadoc. Some javadoc.
+ * @version 1.0
+ * @deprecated Some javadoc. Some javadoc.
+ * @see Some javadoc.
+ * @author max Some javadoc.
+ */
+class InputFormattedJavaDocTagContinuationIndentation implements Serializable {
+  /**
+   * The client's first name.
+   *
+   * @serial Some javadoc. Some javadoc.
+   */
+  private String firstName;
+
+  /**
+   * The client's first name.
+   *
+   * @serial Some javadoc.
+   */
+  private String secondName;
+
+  /**
+   * The client's first name.
+   *
+   * @serialField secondName String
```

Comment:
It is not a typo, right? You intentionally put `secondName` which is a different field name from the one this java doc belongs to.

Reply:
We need some real identifier for javadoc tool.
For checkstyle and formatter this does not matter much.

Reply:
Looks at https://docs.oracle.com/en/java/javase/17/docs/specs/serialization/examples.html